src/js/common/schemaform/save-in-progress/reducers.js

```diff
@@ -123,6 +124,7 @@ export function createSaveInProgressInitialState(formConfig, initialState) {
     prefillStatus: PREFILL_STATUSES.notAttempted,
     isStartingOver: false,
     migrations: formConfig.migrations,
+    prefillTransform: formConfig.prefillTransform,
```

Comment:
Not thrilled about keeping this in the Redux state, but the prop wiring is hard to follow otherwise.

Reply:
If this is a function, may want to rename it if it's going on the state e.g. `prefillTransformer`. Otherwise this should be fine as long as we don't plan on serializing the state.

Reply:
That makes sense. And yeah, we're not planning on serializing the state. There are other functions in there already and we only currently serialize the `data` part (everything else can be rehydrated from the config file or isn't something we'd save).